pkg/devcontainers/features/oci.go

```
@@ -0,0 +1,370 @@
+		return nil, "", err
+	}
+	req.Header.Set("Accept", "application/vnd.oci.image.manifest.v1+json")
+
+	resp, err := client.Do(req)
+	if err != nil {
+		return nil, "", err
+	}
+
+	var token string
+	if resp.StatusCode == http.StatusUnauthorized {
+		resp.Body.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)